backend/app/schemas/reserved_env_vars.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class ReservedEnvVars(str, Enum):
```

Comment:
No need to create another module. This can be moved to `schemas/experiment_template.py`